ereuse_devicehub/static/js/main_inventory.js

```diff
@@ -317,6 +317,29 @@ function export_file(type_file) {
     }
 }
 
+class lotsSearcher {
+    static lots = [];
+
+    /**
+     * do search when lot change in the search input
+     */
+    static doSearch(inputSearch) {
+        const lotsList = document.getElementById("LotsSelector").children;
+
+        for (let i = 0; i < lotsList.length; i++) {
+            const lot = lotsList[i].querySelector("label");
+            if (lot.innerText.toLowerCase().includes(inputSearch.toLowerCase())) {
+                lot.parentElement.style.display = "";
+            } else {
+                lot.parentElement.style.display = "none";
+            }
+        }
+    }
+}
+
+document.addEventListener("DOMContentLoaded", () => {
+    document.getElementById("lots-search").addEventListener("input", (e) => { lotsSearcher.doSearch(e.target.value) })
```

Comment:
May this code fail if `document.getElementById("lots-search")` returns `null`?

See #278 for a related example.

Reply:
Fixed here -->191539608b2d7d560ddea64676b84e459af13800